.circleci/config.yml

```diff
@@ -402,7 +402,7 @@ workflows:
             - Setup Environment
           matrix:
             parameters:
-              dockerimageflag: [ "native:ga", "native:maintenance", "native:dev", "from-yml" ]
+              dockerimageflag: [ "native:ga", "native:maintenance", "native:dev", "native:candidate", "from-yml" ]
```

Comment:
since @YuvHayun should be merging shortly lets add this change in in a seperate pr into the gilab file